database/index.js

```diff
@@ -54,6 +54,16 @@ var addTask = function(data, callback)
 	});
 }
 
+var addProject = function(data, callback)
+{
```

Comment:
General rules for complex (compound) statements:
1- Put the opening bracket at the end of the first line.
2- Use one space before the opening bracket.
3- Put the closing bracket on a new line, without leading spaces.
var addProject = function(data, callback) {
}